# Microbial-Genomics-project
Project for the Computational Microbial Genomics course, led by Professor Segata, academic year 2023/2024.
## Motivation and General Workflow

### Motivation
This report seeks to obtain comprehensive information about a dental plaque SGB, using a set of 30 genomes (MAGs). Our primary objective is to determine if peri-implantitis can be characterized based on the bacterial composition in different plaque samples. Additionally, we will compare our findings with patient metadata to identify external factors, such as smoking, obesity, and sex, that might impact the progression of the disease. The study involves a detailed analysis of the bacterial genomes and their assembly.

### Workflow
1. **Sample Collection and Sequencing:** Bacterial genomes were collected from the oral cavities of Italian patients after dental implants were placed. High-throughput sequencing machines were then used to sequence the genomes.

2. **Analysis Pipeline:**
   1. **Taxonomic Assignment:** Classify the genomes to identify the specific bacterial species present.
   2. **Genome Annotation:** Annotate the genomes to reveal their functional features.
   3. **Pangenome Analysis:** Identify shared and unique genes across the samples through pangenome analysis.
   4. **Phylogenetic Analysis:** Construct a phylogenetic tree to understand evolutionary relationships among the genomes.
   5. **Association with Host Metadata:** Correlate bacterial data with host metadata to investigate the association between peri-implantitis and external factors.
